community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md

```diff
@@ -50,6 +50,32 @@ This creates a controller node with the following attributes:
 For a complete example using this module, see
 [slurm-gcp-v5-cluster.yaml](../../../examples/slurm-gcp-v5-cluster.yaml).
 
+### Reconfigure Option
```

Comment:
I this the feature official name? was that already defined somewhere?  Is it clear? 
Otherwise, we could maybe call it Live Cluster Reconfiguration (`enable_reconfigure`)

Reply:
Updated